lib/store/cleanup_test.go

```diff
@@ -221,41 +226,143 @@ func TestCleanupManageDiskUsage(t *testing.T) {
 	state, op, cleanup := fileOpFixture(clk)
 	defer cleanup()
 
-	for i := 0; i < 100; i++ {
+	s, err := size(op)
+	require.Nil(err)
+	require.Equal(int64(0), s)
+
+	for i := 0; i < 10; i++ {
 		require.NoError(op.CreateFile(core.DigestFixture().Hex(), state, 5))
 	}
 
-	usage, err := m.scan(op, time.Hour, time.Hour)
-	require.NoError(err)
-	require.Equal(int64(500), usage)
+	s, err = size(op)
+	require.Nil(err)
+	require.Equal(int64(50), s)
 }
 
-func TestCleanupManagerAggressive(t *testing.T) {
+func TestCleanupManagerDiskMetrics(t *testing.T) {
 	require := require.New(t)
 
+	clk := clock.New()
 	config := CleanupConfig{
-		AggressiveThreshold: 80,
-		TTL:                 10 * time.Second,
-		AggressiveTTL:       5 * time.Second,
+		TTI: 1 * time.Hour,
+		TTL: 1 * time.Hour,
 	}
 
-	clk := clock.NewMock()
 	m, err := newCleanupManager(clk, tally.NoopScope)
 	require.NoError(err)
 	defer m.stop()
 
-	_, op, cleanup := fileOpFixture(clk)
+	state, op, cleanup := fileOpFixture(clk)
 	defer cleanup()
 
-	require.Equal(m.checkAggressiveCleanup(op, config, func() (int, error) {
-		return 90, nil
-	}), 5*time.Second)
+	for i := 0; i < 100; i++ {
+		require.NoError(op.CreateFile(core.DigestFixture().Hex(), state, 5))
+	}
+
+	testStats := tally.NewTestScope("", map[string]string{})
+	m.clean(testStats, config, op)
+	snapshot := testStats.Snapshot()
+	usageGauge, ok := snapshot.Gauges()["disk_usage+"]
```

Comment:
Not sure where the `+` comes from. Could this be a bug?

Reply:
no it's just how tally does the metrics names, in the monorepo we have a tallytest library that takes care of these quirks